CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.2.2] - 2018-11-03
```

Comment:
@thenewwazoo The link definition for the changes between v0.2.1 and v0.2.2 is missing.